### PR TITLE
Preserve default config for LLM

### PR DESCRIPTION
**Pull request by Wovchena**
Tickets: CVS-156659, CVS-149440

### CODE REVIEW

**Comment by Wovchena** [comment]
@as-suvorov, @popovaan, there's probably similar mistake in other bindings

**Comment by as-suvorov** [comment]
> @as-suvorov, @popovaan, there's probably similar mistake in other bindings

Whisper pipeline looks correct: https://github.com/openvinotoolkit/openvino.genai/blob/master/src/python/py_whisper_pipeline.cpp#L204

**Review by sbalandi — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Could you please also fix
```
The macOS-12 environment is deprecated, consider switching to macOS-13, macOS-14 (macos-latest) or macOS-15. For more details, see https://github.com/actions/runner-images/issues/10721
```
which prevents from merge?

**Review by AlexKoff88 — APPROVED** [review]

File: tools/who_what_benchmark/whowhatbench/wwb.py [hunk 1]
```diff
@@ -385,11 +384,7 @@ def diff_strings(a: str, b: str, *, use_loguru_colors: bool = False) -> str:
 
 
 def genai_gen_answer(model, tokenizer, question, max_new_tokens, skip_question):
-    config = openvino_genai.GenerationConfig()
-    config.max_new_tokens = max_new_tokens
-    config.do_sample = False
-    out = model.generate(question, config)
-    return out
+    return model.generate(question, do_sample=False, max_new_tokens=max_new_tokens)
```
Comment (AlexKoff88) [inline tools/who_what_benchmark/whowhatbench/wwb.py:387]: Can we do the same thing in the samples we have?

Comment (Wovchena) [reply tools/who_what_benchmark/whowhatbench/wwb.py:387]: The samples intentionally avoid this to get more deterministic results depending only on GenAi defaults.

Comment (AlexKoff88) [reply tools/who_what_benchmark/whowhatbench/wwb.py:387]: Thanks, it is fair enough. But how do we educate the user to get the same generation results when they transition from HF API to OV GenAI which is likely to happen?

Comment (Wovchena) [reply tools/who_what_benchmark/whowhatbench/wwb.py:387]: Fair point :) @ilya-lavrenov, can I update the samples to preserve models' defaults and  freeze `max_new_tokens` and `do_sample` only?

Comment (ilya-lavrenov) [reply tools/who_what_benchmark/whowhatbench/wwb.py:387]: > Fair point :) @ilya-lavrenov, can I update the samples to preserve models' defaults and freeze max_new_tokens and do_sample only?

Agree

Comment (Wovchena) [reply tools/who_what_benchmark/whowhatbench/wwb.py:387]: Sure. I just can't pass tests here

**Pull request merged**